Still getting : 
```
test_fft_poisson_3d.py:65: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
fft_poisson_3d.py:33: in fft_poisson
    rho_hat = af.fft3(rho)
../../../anaconda3/lib/python3.6/site-packages/arrayfire/signal.py:223: in fft3
    c_dim_t(dim0), c_dim_t(dim1), c_dim_t(dim2)))
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

af_error = 202

    def safe_call(af_error):
        if (af_error != ERR.NONE.value):
            err_str = c_char_ptr_t(0)
            err_len = c_dim_t(0)
            backend.get().af_get_last_error(c_pointer(err_str), c_pointer(err_len))
>           raise RuntimeError(to_str(err_str))
E           RuntimeError: In function void opencl::verifySupported(af::dim4) [with int rank = 3]
E           In file src/backend/opencl/fft.cpp:68
E           Invalid argument at index 1
E           Expected: isSupLen(dims[i])

../../../anaconda3/lib/python3.6/site-packages/arrayfire/util.py:79: RuntimeError
------------------------------------------------------------ Captured stdout call -------------------------------------------------------------
N_x 66
N_y 76
N_z 48
dx 0.0454545454545
dy 0.0657894736842
dz 0.0833333333333
========================================================== 1 failed in 1.33 seconds ===========================================================
```

--- a/test_fft_poisson_3d.py
+++ b/test_fft_poisson_3d.py
@@ -23,9 +23,9 @@ def test_fft_poisson():
     y_end = np.random.randint(1, 4)
     z_end = np.random.randint(1, 4)
 
-    N_x = np.random.choice(2 * np.arange(20, 40))
-    N_y = np.random.choice(2 * np.arange(20, 40))
-    N_z = np.random.choice(2 * np.arange(20, 40))
+    N_x = np.random.choice(2 ** np.arange(5, 10))
+    N_y = np.random.choice(2 ** np.arange(5, 10))
+    N_z = np.random.choice(2 ** np.arange(5, 10))
     
     dx = (x_end - x_start) / N_x
     dy = (y_end - y_start) / N_y

